test/ClangImporter/availability_open_enums_macos_swift3.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify -I %S/Inputs/custom-modules -swift-version 3 -target x86_64-apple-macosx10.13 %s 
```

Comment:
Hmm... that's a good point. I think I can drop the specific target in the RUN line and fuzzy-match the output to get it to run on all platforms. That would make me much more comfortable.